app/components/SongCard/index.js

```diff
@@ -138,41 +137,52 @@ export function SongCard({ song, trackDetails, width, height, onActionClick }) {
       <IconsContainer>
         <ControlButton
           data-testid="play-btn"
-          onClick={() => handleMusic(actions.PLAY, previewUrl)}
-          disabled={play}
-          type={play ? 'text' : 'ghost'}
-          icon={<PlayCircleTwoTone style={commonIconStyle} />}
+          onClick={handleMusic}
+          disabled={isPlaying()}
+          type={isPlaying() ? 'text' : 'ghost'}
```

Comment:
What is this type?

Reply:
Button type, that changes according to disable, enable states when a song is played or stopped.